admin/views/config-page.php

```diff
@@ -42,15 +42,45 @@
 					<tr valign="top">
 						<th scrope="row">
 							<label
-								for="mincomlengtherror"><?php _e( 'Error message', 'yoast-comment-hacks' ); ?></label>
+								for="mincomlengtherror"><?php _e( 'Error message for comment that is too short', 'yoast-comment-hacks' ); ?></label>
 						</th>
 						<td>
-					<textarea rows="4" cols="100"
+					<textarea rows="4" cols="80"
 					          name="<?php echo esc_attr( $this->option_name ); ?>[mincomlengtherror]"
 					          id="mincomlengtherror"><?php echo esc_html( $this->options['mincomlengtherror'] ); ?></textarea>
 						</td>
 					</tr>
 				</table>
+
+                <h3><?php _e( 'Maximum comment length', 'yoast-comment-hacks' ); ?></h3>
+
+                <p><?php _e( 'Users that try to submit a comment longer than the length you set below will get an error immediately. The text of that error is specified below too.', 'yoast-comment-hacks' ); ?></p>
+                <table class="form-table">
+                    <tr valign="top">
+                        <th scrope="row">
+                            <label
+                                    for="maxcomlength"><?php _e( 'Maximum length', 'yoast-comment-hacks' ); ?>
+                            </label>
+                        </th>
+                        <td>
+                            <input type="number" class="small-text" min="5" max="255"
```

Comment:
A maximum of 255 characters might feel restricted for certain websites.
I assume this is based on SEO specifications, if so I would opt for adding a little contextual explanation about why this maximum value restriction is chosen.